src/sentry/options/defaults.py

```diff
@@ -109,10 +109,18 @@
 register('symbolserver.enabled', default=False, flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK)
 register(
     'symbolserver.options',
-    default={'url': 'http://127.0.0.1:3000'},
+    default={'url': 'http://127.0.0.1:3021'},
     flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK
 )
 
+# Symbolicator
+register('symbolicator.enabled', default=False, flags=FLAG_ALLOW_EMPTY | FLAG_PRIORITIZE_DISK)
+register(
+    'symbolicator.options',
+    default={'url': 'http://127.0.0.1:'},
```

Comment:
Is this intentional with a trailing `:` ?

Reply:
I don't think so, fixing it